### ISSUE TITLE
Downstream Pipelines not scheduled after successful Manually Triggered Pipeline build

### ISSUE THREAD

**Issue by gavingmiller** [gocd/gocd#4981]
##### Issue Type

- Bug Report

##### Summary

We've experienced a situation where our downstream, fan-out pipelines won't auto-schedule when the current pipeline (pictured below) has been manually triggered.

![screen shot 2018-07-25 at 3 26 57 pm](https://user-images.githubusercontent.com/452161/43228686-7bed44b8-901f-11e8-92e3-6720edbc85e2.png)

##### Environment

skipped. available if required

###### Basic environment details

* Go Version: `18.7.0 (7121-75d1247f58ab8bcde3c5b43392a87347979f82c5)` (updated with latest)
* JAVA Version: `1.8.0_181`  (updated with latest)
* OS: `Linux 2.6.32-573.7.1.el6.x86_64`  (updated with latest)

##### Steps to Reproduce

Not available at this time.

##### Expected Results

When manually triggered a pipeline build, I would expect the downstream builds to trigger as well once the pipeline completes successfully.

##### Actual Results

The triggered pipeline builds and runs to completion and the downstream pipelines do not trigger.

##### Log snippets

Unable to find any log snippets that appear relevant. Our original guess was a skewed clock on the go master server, but this issue still occurred once we resolved that issue.

**Comment by ketan** [comment]
Mind clicking on the label 755, on the pipeline and pasting a screenshot?

**Comment by arvindsv** [comment]
15.2.0 is more than 3 years old. It's going to be difficult to help. Just saying.

**Comment by gavingmiller** [comment]
@arvindsv you're not wrong! :hurtrealbad: if it makes life easier, I can close this bug, upgrade to see if this continues to happen, and reopen if so.

@ketan the 755 link is just an empty anchor `#`. Are you looking for the value stream map?

**Comment by ketan** [comment]
> @arvindsv you're not wrong!  if it makes life easier, I can close this bug, upgrade to see if this continues to happen, and reopen if so.

We'd prefer it if you do that.

> @ketan the 755 link is just an empty anchor #. Are you looking for the value stream map?

Yes, please vsm for 755 with the latest gocd version, please.

**Comment by gavingmiller** [comment]
@ketan @arvindsv Reopening issue since we're still experiencing this issue after upgrade. Here's a screenshot of the VSM:

![screen shot 2018-08-03 at 11 35 26 am](https://user-images.githubusercontent.com/452161/43657122-0a2e6740-9712-11e8-845f-dcc4e4425110.png)

![screen shot 2018-08-03 at 11 36 08 am](https://user-images.githubusercontent.com/452161/43657128-0d7582bc-9712-11e8-804e-c3c2a8cb6e80.png)



New server/version information (updated in original issue too):

```
Go Server Version:	18.7.0 (7121-75d1247f58ab8bcde3c5b43392a87347979f82c5)
JVM version:	1.8.0_181
OS Information:	Linux 2.6.32-573.7.1.el6.x86_64
Usable space in artifacts repository:	103 GB
Database schema version:	1807002
Pipelines Count:	96
```

We triggered the pipeline manually to start with and it was successfully picked up and scheduled by the downstream jobs. But then when we triggered the build with a commit, it was not picked up.
